script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2B.txt

```diff
@@ -91,16 +91,16 @@ With that in mind, I look at Arcueid's face―――
 We're not talking about how to avoid death, we're talking about how to kill."
 <Page29>
 "Well that's―――true I guess."
-"You're going against a vampire from now on.
+"You're going up against a vampire from now on.
 That's why you must throw away your sense of morality, even if it's just for tonight. Carrying around that burden will only weigh you down when the time comes."
 <Page30>
 "...I know that much. Our opponent is a monster after all, which is why I'm willing to help."
 <Page31>
 "No, you don't understand at all, Shiki.
-Cutting off a limb? You should give up on such a suicidal act. If you have time to cut off his limbs, then you should start by cutting away his life force. It'd be different if you were any other creature, but this is something only you can do.
-Listen. Don't give Vlov a chance to fight back. The only advantage you have is that he doesn't know about your ability."
+Cutting off a limb? You should give up on such a suicidal act. If you have time to cut off his limbs, you should use that time to sever his life instead. It would be different if he were any other creature, but as it is this is something only you can do.//Is this the correct association of the last sentence? 他の生き物ならともかく、あなたにだけはそれが出来るんだから。
```

Comment:
I think this version makes more sense so I'm going to assume it's right. The only source of doubt is that the subject of the past couple of sentences is definitely Shiki.